F/Flex/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/westes/flex.git", "d69a58075169410324fe49666f6641ba6a9d1f91")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flex
+apk add texinfo
+apk add help2man
+./autogen.sh
+./configure --prefix=${prefix}
+make -j${nprocs}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc="glibc"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("i686", "linux"; libc="musl"),
+    Platform("x86_64", "linux"; libc="musl"),
+]
```

Comment:
Can we make this do all `supported_platforms()`?

Reply:
`texinfo` and `help2man` don't have _jll counterparts. So currently I've apk added them. And also crosscompiling doesn't work for Linux x86_64 also. Currently these are the only platforms it can build. Soon I'll add for the rest.